src/master/grafana/dashboards/InfiniBand/network-counters.json

```diff
@@ -658,22 +785,21 @@
         "skipUrlSync": false,
         "sort": 1,
         "tagValuesQuery": "",
-        "tags": [],
         "tagsQuery": "",
         "type": "query",
         "useTags": false
       },
       {
-        "allValue": null,
         "current": {
-          "selected": false,
-          "text": "node-000",
-          "value": "node-000"
+          "selected": true,
+          "text": "ip-AC1A0008",
+          "value": "ip-AC1A0008"
```

Comment:
why do we need this change?

Reply:
Yea let me address this. I exported this from Grafana. I can change back but I imagine the node-000 came from an earlier export.

Reply:
Addressed. FYI this default value is never seen on the portal because it is replaced by the real hostname for the instances